internal/credentials/credentials_test.go

```diff
@@ -139,18 +139,20 @@ var _ = Describe("Credentials", func() {
 		Expect(err).NotTo(HaveOccurred())
 	})
 	It("EnsureAdminPasswordSecret already exists", func() {
-		passwordHash, err := cm.EnsureAdminPasswordSecret(ctx, clusterDeployment)
+		passwordHash, err := cm.EnsureAdminPasswordSecret(ctx, clusterDeployment, "")
 		Expect(err).NotTo(HaveOccurred())
 		// Verify the password secret
 		passwordSecret := &corev1.Secret{}
 		err = cm.Client.Get(ctx, client.ObjectKey{Namespace: clusterDeployment.Namespace, Name: clusterDeployment.Name + "-admin-password"}, passwordSecret)
 		Expect(err).NotTo(HaveOccurred())
 		password, exists := passwordSecret.Data["password"]
 		Expect(exists).To(BeTrue())
-		// Call again, the password hash should change but the password shouldn't
```

Comment:
Is there any reason we wanted this behavior previously?
@eranco74 ?

Reply:
What do you mean?
What behavior?

Reply:
Right, but we didn't *need* the hash to change for any reason, right?

Reply:
We talked with @eranco74 today and he didn't remember any reason for it